pkg/bpf/map_linux.go

```diff
@@ -981,7 +981,12 @@ func (m *Map) Delete(key MapKey) error {
 		metrics.BPFMapOps.WithLabelValues(m.commonName(), metricOpDelete, metrics.Errno2Outcome(errno)).Inc()
 	}
 	if errno != 0 {
-		err = fmt.Errorf("unable to delete element %s from map %s: %w", key, m.name, errno)
+		if errno == 2 {
+			// ignore errno 2 "no such file or directory"
+			m.scopedLogger().WithField("element", key).WithField("map", m.name).Warn("element not found")
```

Comment:
```suggestion
			m.scopedLogger().WithFields(logrus.LogFields{
			    "key": key,
			    "map": m.name,
			}).Warn("element not found")
```

It's worth seeing if we already having `logfields` for each field in the above entry, i.e. `logfields.MapKey`, `logfield.MapName`, etc.